### DISCUSSION TITLE
Wrapper to integrate vsomeip library to a C project

### DISCUSSION THREAD

**Discussion by RobertoGeografo** [COVESA/vsomeip#790] [Q&A] [answered]
Hi everyone,

Hopefully I am asking this question in the right place. I am working on integrating the vsomeip library to a C project.

Is there a Wrapper, C API or anything already built that I can use for the integration? Just thought of asking before writing something myself.

Thanks for your support.

Best regards,

Rob

**Comment by duartenfonseca** [comment, marked as answer]
Hi, there is no wrapper for C.
